static/js/stream_data.js

```diff
@@ -7,6 +7,9 @@ var exports = {};
 // Call clear_subscriptions() to initialize it.
 var stream_info;
 var subs_by_stream_id;
+var neversubbed_streams_map = _.object(_.map(page_params.neversubbed_info, function(stream) {
```

Comment:
Probably this should be shifted to more appropriate place. @neerajwahi any suggestions?